cmd/bosun/sched/sched.go

```diff
@@ -589,13 +589,6 @@ func (s *Schedule) Run() error {
 		go s.PingHosts()
 	}
 	go s.Poll()
-	go func() {
-		// Sleep before starting unknown checks. If Bosun was down for a little
-		// while then all the state.Touched times will be old and a bunch of
-		// unknowns will go off because it hasn't run all the checks yet.
-		time.Sleep(s.Conf.CheckFrequency)
```

Comment:
This feature needs to be added back. This is because if bosun is down for 10 minutes and scollectors all around have been queueing data, it will take a few minutes for that data to get back into bosun. Thus, when rules are run, they won't return anything, and then get marked as unknown.